src/matrixrtc/NewMembershipManagerActionScheduler.ts

```diff
@@ -111,13 +115,13 @@ export class ActionScheduler {
                     this._actions.push(...actionUpdate.insert);
                 }
             }
+            this.running = false;
         } catch (e) {
             // Set the rtc session "not running" state since we cannot recover from here and the consumer user of the
             // MatrixRTCSession class needs to manually rejoin.
             this.running = false;
             throw e;
         }
```

Comment:
```suggestion
        } finally {
            this.running = false;
        }
```

Unless you want to keep the comment?

Reply:
I thinks then I will not set it to false in the throw case?

Reply:
Yes you are totally right this is working and simpler.